apps/heureka/src/hooks/useUrlState.js

```diff
@@ -57,7 +63,7 @@ const useUrlState = () => {
     }
 
     setIsURLRead(true)
-  }, [isURLRead])
+  }, [isURLRead, setActiveView, setFiltersFromURL, setShowPanel, setServiceDetail, setIssueDetail])
```

Comment:
I was a bit confused here because I have understood that if the setter functions are stable and don’t change between renders, you can safely omit them from the dependencies BUT it doesn´t hurt.

Reply:
No, actually they should be listed here, as otherwise the useEffect does not affect on applying filter changes right after selecting them into URL.